libs/components/lookup/src/lib/modules/lookup/lookup-show-more-async-modal.component.ts

```diff
@@ -58,17 +59,27 @@ export class SkyLookupShowMoreAsyncModalComponent implements OnInit, OnDestroy {
 
   #ngUnsubscribe = new Subject<void>();
 
+  #logSvc: SkyLogService;
+
   #offset = 0;
 
   constructor(
     public modalInstance: SkyModalInstance,
     public context: SkyLookupShowMoreNativePickerAsyncContext,
-    changeDetector: ChangeDetectorRef
+    changeDetector: ChangeDetectorRef,
+    logSvc: SkyLogService
   ) {
     this.#changeDetector = changeDetector;
+    this.#logSvc = logSvc;
   }
 
   public ngOnInit(): void {
+    if (this.context.idProperty === undefined) {
+      this.#logSvc.error(
+        "The 'idProperty' input is required when 'searchAsync' is used."
```

Comment:
You might need to prefix this error message with the name of the component since it won't have much context when it's logged in the console. Maybe, `The lookup component requires a value for the 'idProperty' input when the 'searchAsync' output is used.`

Reply:
Done